slamd/discovery/processing/algorithms/discovery_experiment.py

```diff
@@ -115,6 +119,37 @@ def _preprocess_features(self):
                 self.features_df.loc[:, feature] = encoder.fit_transform(self.features_df[[feature]])
         self.features_df = self.features_df.dropna(axis=1)
 
+    def _movecol(self, df, cols_to_move=[], ref_col='', place='after'):
+        """
+        Move one or several columns before or after a given reference column.
+        Adapted from: https://towardsdatascience.com/reordering-pandas-dataframe-columns-thumbs-down-on-standard-solutions-1ff0bc2941d5
+        """
+        cols = df.columns.tolist()
+        if place == 'after':
+            seg1 = cols[:list(cols).index(ref_col) + 1]
+            seg2 = cols_to_move
+        elif place == 'before':
```

Comment:
We only use the 'after' version so far in our code. Therefore we should at the moment not include the 'before' scenario. This way, we do not need the place parameter. Also we only use 'Row number' as reference column. So I think we should get rid of this param as well. Then we can be more specific and might call the method some like 'move_after_row_column'